dpnp/dpnp_iface_manipulation.py

```diff
@@ -1153,19 +1351,14 @@ def result_type(*arrays_and_dtypes):
 
     Parameters
```

Comment:
no, because it suppress a possible warning generated by `flake8` here (the 1st string in the docstring must be one-lined string).
in the generated documentation the line `result_type(*arrays_and_dtypes)` is a signature of the function